app/src/modules/settings/routes/data-model/new-collection.vue

```diff
@@ -317,15 +317,15 @@ export default defineComponent({
 						options: {
 							choices: [
 								{
-									text: t('Published'),
+									text: t('status_field.published'),
 									value: 'published',
 								},
 								{
-									text: t('Draft'),
+									text: t('status_field.draft'),
 									value: 'draft',
 								},
 								{
-									text: t('Archived'),
+									text: t('status_field.archived'),
 									value: 'archived',
 								},
```

Comment:
This will save the output of the `t` function to the database, which would still "lock" it to whatever language the admin user was using at the time of setting up the collection 🤔